t/sql/mozart.sql

```diff
@@ -0,0 +1,188 @@
+-- Automatically generated, do not edit.
```

Comment:
Right, I ran the dump-entities-sql script on astro, so it's dumped from the live database. It's a much easier way to generate test data when you know there's already an entity in the database that has certain characteristics. Otherwise the test data would have to be tediously written by hand.

For where it gets used in a test, see e.g. https://github.com/metabrainz/musicbrainz-server/pull/110/files#diff-c8e07cb07658abf5d9f49ff27878e7b1R251 The second argument to `prepare_test_database` specifies a file from t/sql/ to load.